gsdocs/guideline.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "HSF GSoC Mentor Guidelines"
+author: "Benedikt Hegner"
+layout: default
+---
+
+# How to Add a New project in HSF GSDocs
+
+## Instructions for Adding a New Proposal
+
+ * Option A: email GSoC [administrators](mailto:hsf-gsdocs-admin@googlegroups.com) (currently: Andrei Gheata, Antoine Pérus and Xavier Valls)
```

Comment:
Should this still be an option? Shouldn't we do everything over github?

Reply:
Not sure, time is short, we need to consider also things sent by mail